.github/workflows/build.yml

```
@@ -22,8 +22,17 @@ jobs:
             node-version: '14.x'
             primary: true
     runs-on: ${{ matrix.os }}
+    outputs:
+      version: ${{ steps.output_version.outputs.version }}
     steps:
     - uses: actions/checkout@v2
+    - name: Extract version
+      id: extract_version
+      uses: Saionaro/extract-package-version@v1.1.1
+    - name: Print version
+      run: echo ${{ steps.extract_version.outputs.version }}
+    - id: output_version
+      run: echo "::set-output name=version::${{ steps.extract_version.outputs.version }}"
```

For such a simple task, we can use:


```suggestion
    - id: package_info
      run: |
        VERSION=$(node -e 'console.log(require("./package.json").version)')
        echo "::set-output name=version::$VERSION"
```